unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -147,6 +158,38 @@ function checkArguments() {
 
 }
 
+function checkTools() {
+
+    # -------------- Check tools required to run the script (awk, sed, etc.) -----------------------------------------
+
+    toolList=(  "awk" "cat" "chown" "cp" "curl" "echo" "export"
+                "free" "grep" "kill" "mkdir" "mv" "rm" "sed"
+                "sudo" "tar" "touch" "uname")
+
+    missingtoolsList=()
+    for command in "${toolList[@]}"
+    do
+        if [ -z "$(command -v $command)" ]; then
```

Comment:
```suggestion
        if [ -z "$(command -v ${command})" ]; then
```
Any problem with that?

Reply:
Nop, Done